salt/modules/yumpkg.py

```diff
@@ -258,7 +259,8 @@ def _get_extra_options(**kwargs):
     Returns list of extra options for yum
     '''
     ret = []
-    for key, value in kwargs.items():
+    kwargs = salt.utils.clean_kwargs(kwargs)
```

Comment:
You need to use the double-asterisk here. See other uses of this function in the codebase for reference.